ceph.spec.in

```diff
@@ -488,8 +488,14 @@ BuildArch:      noarch
 Group:          System/Filesystems
 %endif
 Requires:       ceph-mgr = %{_epoch_prefix}%{version}-%{release}
+%if 0%{?fedora} || 0%{?rhel}
 Requires:       numpy
 Requires:       scipy
+%endif
+%if 0%{?suse_version}
```

Comment:
yeah, this is more correct! but i think we can do better, like:

```
%if 0{?fedora} || 0%{?suse_version}
Requires:       python%{_python_buildid}-numpy
%if 0{without python2}
Requires:       python3-scipy
%else
Requires:       python2-scipy
%endif
%endif
%if 0%{?rhel}
Requires:       numpy
Requires:       scipy
%endif
```

see https://rpmfind.net/linux/rpm2html/search.php?query=python2-scipy&submit=Search+...&system=&arch=
and https://rpmfind.net/linux/rpm2html/search.php?query=python-scipy&submit=Search+...&system=&arch=